wys/api/python/wys_api.py

```diff
@@ -461,7 +461,8 @@ def update_locations(conn, loc_table):
         """
         cur.execute(update_locations_sql)
 
-def get_schedules(conn, api_key):
+def get_schedules(conn, api_connection):
+    api_key = api_connection.password
```

Comment:
this should be moved back to the dag file but inside the decorated task function